pkg/monitor/cluster/cluster.go

```diff
@@ -68,11 +76,16 @@ func (mon *Monitor) Monitor(ctx context.Context) error {
 	mon.log.Debug("monitoring")
 
 	// If API is not returning 200, don't need to run the next checks
-	statusCode, err := mon.emitAPIServerHealthzCode(ctx)
+	statusCode, err := mon.emitAPIServerHealthzCode()
 	if err != nil || statusCode != http.StatusOK {
 		return err
 	}
 
+	err = mon.emitNodesMetrics()
```

Comment:
see https://github.com/Azure/ARO-RP/pull/377#discussion_r401145069 - please sync with @sjkingo

Reply:
but my code is after the healthz call. I don't quite understand what I need to do in relation to the comment you mentioned.

Reply:
please update `func (mon *Monitor) Monitor(ctx context.Context) error` -> `func (mon *Monitor) Monitor(ctx context.Context)` (remove the error return).

If emitAPIServerHealthzCode fails, log the error if applicable and return early

If any subsequent check fails, log the error but continue regardless.

Reply:
yes, just pushed. 
does it make sense to also update `emitPrometheusAlerts` to follow the same logic?